cogs/games.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import os
+
+from discord import Member, Embed
+from discord.ext.commands import Cog, command
+from dotenv import load_dotenv
+
+from entities.members import MemberDto
+
+load_dotenv()
+
+
+class Games(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @Cog.listener()
+    async def on_ready(self):
+        if not self.bot.ready:
+            self.bot.cogs_ready.ready_up(os.path.basename(__file__))
+
+    @command(name='barbut', pass_context=True)
```

Comment:
Add description to command: description='Description'

Reply:
Ce e asta? ca nu inteleg

Reply:
Resolved on discord